servers/visual/visual_server_raster.cpp

```diff
@@ -98,7 +98,8 @@ void VisualServerRaster::draw(bool p_swap_buffers, double frame_step) {
 	//needs to be done before changes is reset to 0, to not force the editor to redraw
 	VS::get_singleton()->emit_signal("frame_pre_draw");
 
-	changes = 0;
+	changes[0] = 0;
+	changes[1] = 0;
```

Comment:
Could be `changes = {};` I think?

Reply:
From a quick check in godbolt, although that works for initialization it doesn't seem to work dynamically for assignment. So google suggests memset or a loop, both of which are probably overkill for two elements. (Unless there's another shortcut for this).